### PR TITLE
Capstone - Microverse

### PR DESCRIPTION
**Pull request by iwillteixeira**
### Capstone - microverse - module 1

- Create a mobile version.
- Create queries for the desktop version.
- Add dynamically items with js.
- Fix Linters errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,66 @@
...
+- CSS
+- JS
+- Bootstrap 5
+
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+- [Live Demo](https://iwillteixeira.github.io/microverse-capstone/index.html)
+- [Video](https://iwillteixeira.github.io/microverse-capstone/index.html)
```
Comment (carlos-ssh) [inline README.md]: - **Kindly provide a video link that works because this link sends you to the live demo project**

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @iwillteixeira, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


changesrequested



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (iwillteixeira) [reply README.md]: Hello Carlos!

Thanks for reviewing my project, I just fix the README.md with the right link to the video.

With gratitude Will.

**Review by carlos-ssh — APPROVED** [review]
# 🍾🍾 **STATUS APPROVED** 👏🏽👏🏽

Hi @iwillteixeira , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**